lib/player/player-service.ts

```diff
@@ -13,49 +13,74 @@ export namespace playerService {
     minStartingBid: number;
     averageBuyNow: number;
     averageStartingBid: number;
-    samplecount: number;
+    sampleCount: number;
   }
+
   export async function getMarketPrice(
     resourceId: number
   ): Promise<MarketPrice> {
+    let invalidSamples = 0;
     const auctions = await getAuctions(resourceId);
-
     const price: MarketPrice = {
       minBuyNow: Number.MAX_VALUE,
       minStartingBid: Number.MAX_VALUE,
       averageBuyNow: 0,
       averageStartingBid: 0,
-      samplecount: 0
+      sampleCount: 0
     };
 
     auctions.forEach(a => {
-      if (a.itemData.resourceId != resourceId) return;
+      if (a.itemData.resourceId != resourceId) {
+        invalidSamples++;
+        return;
+      }
+
+      // ignore the "default" 10000 buy-now price
+      if (a.buyNowPrice === 10000) {

```

Comment:
I think we should change this change to a.buyNowPrice === a.itemData.maxPrice
I don't know if it's called maxPrice but there was a field indicating this default buy now price value

Reply:
It's `marketDataMaxPrice`. Changed it 👍